shared/actions/provision.js

```diff
@@ -213,6 +215,27 @@ class ProvisioningManager {
     )
   }
 
+  switchToGPGSignOKHandler = (
+    params: RPCTypes.ProvisionUiSwitchToGPGSignOKRpcParam,
+    response: CommonResponseHandler,
+    state: TypedState
+  ) => {
+    this._stashResponse('keybase.1.provisionUi.switchToGPGSignOK', response)
+    return Saga.all([
+      Saga.put(ProvisionGen.createSwitchToGPGSignOnly({importError: params.importError})),
+      Saga.put(ProvisionGen.createShowGPGPage()),
```

Comment:
we should just make this a param to the existing showGPGPage action. See the other show* actions which also take payloads. Motivation is the flag will only ever get set by this and its only in regards to this one screen